sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -1195,20 +1270,28 @@ class CastChecks extends ExprChecks {
     case TypeEnum.UDT => (udtChecks, sparkUdtSig)
   }
 
+  override def tagAst(meta: BaseExprMeta[_]): Unit = {
+    meta.willNotWorkInAst("AST is not support by cast")
```

Comment:
"is not support by".  This probably should use the same consistent message per comment above, whichever it is.